index.html

```diff
@@ -1934,6 +1834,66 @@
       </section>
       <section>
         <h3>
+          Interface <a><code>PresentationReceiver</code></a>
+        </h3>
+        <pre class="idl">
+          interface PresentationReceiver {
+            readonly attribute Promise&lt;PresentationConnectionList&gt; connections;
+          };
+
+
+</pre>
+        <div class='issue'>
+          <p>
+            The text about "PresentationReceiver object" should probably move
+            to the section about the Presentation interface. This section is
+            about the interface, not its instance (the object).
+          </p>
+          <p>
+            Furthermore, we probably need a better way to explain how
+            PresentationConnectionList is initialized -- how the set of
+            presentations is populated.
```

Comment:
The place to do this is in 6.4.6, "Establishing a presentation connection in a receiving browsing context" which starts monitoring for incoming presentation connections.  The set of presentations for the receiving context would be initialized there I think.

The "set of presentations" data structure in general needs some cleanup, feel free to drop a reference to https://github.com/w3c/presentation-api/issues/193 instead here.